tools/webpack-make

```diff
@@ -190,7 +190,7 @@ function generateDeps(makefile, stats) {
     lines.push("\trm -rf $(" + prefix + "_OUTPUTS) $(" + prefix + "_INSTALL) " + makefile);
     lines.push("all-local:: " + prefix);
     lines.push("\t@true");
-    lines.push("clean-local:: clean-" + prefix);
+    lines.push("distclean-local:: clean-" + prefix);
```

Comment:
`make clean` is never expected to remove distributed files; but of course everyone is entitled to have their own expectations of what `clean` does, so at this point it's bikeshedding zone. Of course if you are in the git checkout, the One True way to really clean up is `git clean -fdx` :-) (And in fact that's what I commonly use).

The main point of this is to unbreak `make clean` when working with release taballs, i. e. packagers and developers. For these it's outright wrong.